resources/common/plugins/template/cvvc-oto-gen/cvvc-oto-gen.js

```diff
@@ -117,136 +112,322 @@ let aliasCountMap = new Map()
     points.push(start)
-    let extras = [cutoffVC.toString()]
+    let extras = [cutoff.toString()]
     let notes = new Notes()
     if (appendTags) {
-        notes.tag = tag
+        notes.tag = "VC"
     }
     let entry = new Entry(sample, thisAlias, start, end, points, extras, notes)
-    aliasCountMap.set(alias, count + 1)
-    push(entry, false)
+    push(entry, "VC")
+}
+
+function pushSoloC(sample, index, alias) {
+    let thisAlias = checkAliasCount(alias, false, true)
+
+    let start = offset + index * beatLength - consLength
+    let ovl = start + consLength/2
+    let preu = start + consLength/2
+    let fixed = start
+    let cutoff = 0 - (consLength)
+    
+    let end = start - cutoff
+    let points = [fixed, preu, ovl]
+    points.push(start)
+    let extras = [cutoff.toString()]
+    let notes = new Notes()
+    if (appendTags) {
+        notes.tag = "Solo C"
+    }
+    let entry = new Entry(sample, thisAlias, start, end, points, extras, notes)
+    push(entry, "VC")
+}
+
+function pushSoloV(sample, index, alias, nextHasConsonant) {
+    let thisAlias = checkAliasCount(alias, false, false)
+
+    let start = offset + index * beatLength + fixBuffer
+    let ovl = start + ovlVC
+    let preu = start + ovlVC
+    let fixed = start
+    let cutoff = 0 - (beatLength - fixBuffer - ovlVC - (nextHasConsonant ? consLength : 0))
+    
+    let end = start - cutoff
+    let points = [fixed, preu, ovl]
+    points.push(start)
+    let extras = [cutoff.toString()]
+    let notes = new Notes()
+    if (appendTags) {
+        notes.tag = "Solo V"
+    }
+    let entry = new Entry(sample, thisAlias, start, end, points, extras, notes)
+    push(entry, "CV")
+}
+
+function pushVV(sample, index, alias, nextHasConsonant) {
+    let thisAlias = checkAliasCount(alias, false, false)
+
+    let start = offset + index * beatLength - 2*ovlVC
+    let ovl = start + ovlVC
+    let preu = ovl + ovlVC
+    let fixed = preu + fixBuffer
+    let cutoff = 0 - (2*ovlVC + beatLength - ovlVC - (nextHasConsonant ? consLength : 0))
+    
+    let end = start - cutoff
+    let points = [fixed, preu, ovl]
+    points.push(start)
+    let extras = [cutoff.toString()]
+    let notes = new Notes()
+    if (appendTags) {
+        notes.tag = "VV"
+    }
+    let entry = new Entry(sample, thisAlias, start, end, points, extras, notes)
+    push(entry, "CV")
+}
+
+function pushVCV(sample, index, alias, nextHasConsonant) {
+    let thisAlias = checkAliasCount(alias, false, false)
+
+    let start = offset + index * beatLength - consLength - 2*ovlVC
+    let ovl = start + ovlVC
+    let preu = ovl + ovlVC + consLength
+    let fixed = preu + fixBuffer
+    let cutoff = 0 - (2*ovlVC + consLength + beatLength - ovlVC - (nextHasConsonant ? consLength : 0))
+    
+    let end = start - cutoff
+    let points = [fixed, preu, ovl]
+    points.push(start)
+    let extras = [cutoff.toString()]
+    let notes = new Notes()
+    if (appendTags) {
+        notes.tag = "vcv"
```

Comment:
`VCV`?

Reply:
This is lowercase so that you can search for "tag:CV" in the entry list without including vcv entries.

Reply:
I see, this is reasonable.
Seems we need more development on the filter feature 😢